app-backend/db/src/test/scala/com/azavea/rf/database/CampaignDaoSpec.scala

```diff
@@ -1076,4 +1077,65 @@ class CampaignDaoSpec
       }
     }
   }
+
+  test("update campaign projects statuses should update campaign status") {
+    check {
+      forAll {
+        (
+            userCreate: User.Create,
+            campaignCreate: Campaign.Create,
+            annotationProjectCreateOne: AnnotationProject.Create,
+            annotationProjectCreateTwo: AnnotationProject.Create
+        ) =>
+          {
+            val updateCampaignProjectIO = for {
+              user <- UserDao.create(userCreate)
+              insertedCampaign <- CampaignDao
+                .insertCampaign(
+                  campaignCreate.copy(parentCampaignId = None),
+                  user
+                )
+              insertedProjOne <- AnnotationProjectDao.insert(
+                annotationProjectCreateOne.copy(
+                  status = AnnotationProjectStatus.Waiting,
+                  campaignId = Some(insertedCampaign.id)
+                ),
+                user
+              )
+              insertedProjTwo <- AnnotationProjectDao.insert(
+                annotationProjectCreateTwo
+                  .copy(
+                    status = AnnotationProjectStatus.Waiting,
+                    campaignId = Some(insertedCampaign.id)
+                  ),
+                user
+              )
+              _ <- AnnotationProjectDao.update(
```

Comment:
I think to show the update it would be good to get the `campaign` before updating the projects and have an expectation that it should have two waiting projects.

Also, since you're deleting this project anyway and it doesn't affect the expectation, is it correct that the test is the same without this query?